hack/lib/serverless.bash

```diff
@@ -218,16 +218,19 @@ EOF
 
 function ensure_kafka_channel_default {
   logger.info 'Set KafkaChannel as default'
-
+  local defaultChConfig channelTemplateSpec yamls
+  yamls="$(dirname "$(realpath "${BASH_SOURCE[0]}")")/yamls"
+  read.in defaultChConfig < "${yamls}/kafka-default-ch-config.yaml"
+  read.in channelTemplateSpec < "${yamls}/kafka-channel-templatespec.yaml"
```

Comment:
I read the comment on the `read.in` function but honestly don't quite understand the reasoning :sweat_smile:. What's the benefit vs. doing `defaultChConfig=$(cat "${yamls}/kafka-default-ch-config.yaml)`?

Reply:
You're right. Let's simplify that. I just thought it will be necessary...